src/test/java/HttpServer/ResponseWriterTest.java

```diff
@@ -33,4 +33,34 @@ public void itIsNotFoundForFaviconCurl() throws IOException {
         assert(client.received(expectation));
     }
 
+    @Test
+    public void teaIsOkay() throws IOException {
+        MockTraffic traffic = new MockTraffic();
+        MockClient client = new MockClient();
+        traffic.emulateGettingTea();
```

Comment:
I think this `MockTraffic` is masking the fact that it's hard to create request objects in your tests. i.e. All of your tests are creating requests from strings.

On the whiteboard, we discuss how you want to get away from strings ASAP in your code. There should be a similar mindset to creating requests in tests as well.